rs/replicated_state/src/canister_state/system_state.rs

```diff
@@ -125,6 +150,9 @@ pub struct CanisterMetrics {
     pub executed: u64,
     pub interrupted_during_execution: u64,
     pub consumed_cycles: NominalCycles,
+    pub instructions_executed: NumInstructions,
+    pub load_metrics: LoadMetrics,
+
```

Comment:
Personal preference: I don't see any good reason to stick these fields between `consumed_cycles` and `consumed_cycles_by_use_cases`. It would seem more appropriate e.g. to put them after `executed`.

Same with the proto definition.